### DISCUSSION TITLE
Question: Set controls to false on pause on mobile

### DISCUSSION THREAD

**Discussion by byMedia** [vimeo/player.js#672] [General]
Hello,
I have made some customizations to the player to make it look like I want to.

I have made this edit on the pause event:
player.on('pause', function() {
jQuery(".pum- .pum-title.popmake-title").fadeIn();
jQuery(".pum- .play-overlay").fadeIn();
player.setControls(false);
});

But I want to disable the controls on pause. So I added player.setControls(false); to that event. But it didn't work. Would it be possible for you to help me with this ?

And is it possible to only do this on mobile devices ?

Thank you in advance.

**Comment by luwes** [comment]
this is not possible at the moment, `controls` can only be disables by a url param or constructor param. not via an API call. we'll look into adding this in the future. thanks for the feedback.
